newIDE/app/src/EventsSheet/InlineParameterEditor.js

```diff
@@ -108,13 +109,25 @@ export default class InlineParameterEditor extends React.Component<
     const instruction = this.props.instruction;
     if (!instruction) return null;
 
+    const onRequestClose = () => {
+      if (this.state.instructionMetadata)
+        setupInstructionParameters(
+          this.props.globalObjectsContainer,
+          this.props.objectsContainer,
+          instruction,
+          this.state.instructionMetadata,
+          instruction.getParameter(this.props.parameterIndex)
```

Comment:
I think this should be: `objectParameterIndex !== -1 ? instruction.getParameter(objectParameterIndex) : null`

with 
```
const objectParameterIndex = objectName
      ? getObjectParameterIndex(instructionMetadata)
      : -1;
```

because otherwise you risk changing the object name with whatever the parameter is being edited?